### ISSUE TITLE
Investigate standalone getStaticPaths error

### ISSUE THREAD

**Issue by Timer** [vercel/next.js#13832]
See [this tweet](https://twitter.com/femyeda/status/1266065372345753605).

User claims the following error was reported when `getStaticPaths` was defined without a `getStaticProps` function exported:
![image](https://user-images.githubusercontent.com/616428/83936440-a0514a80-a791-11ea-96d6-d1d00cd14dd4.png)

**Comment by anku255** [comment]
@timneutkens @Timer I am taking a look at it. The first thing I will do is to try to reproduce the error.

**Comment by balazsorban44** [comment]
I also started to have a look, and I think the problem might be somewhere here, but I cannot clearly identify it:

https://github.com/vercel/next.js/blob/8bac41284558e2ab860b6b7c1a11c61efcf26266/packages/next/build/utils.ts#L745-L765

**Comment by anku255** [comment]
@Timer I tried to reproduce it but couldn't reproduce the error given in the screenshot. Next.js seems to be displaying the correct error in my testing.

Here is what I tried. 

1. Exported `getStaticPaths` but **not** `getStaticProps`. The name of the file was `post/[id].jsx`

```js


const Post = ({ post }) => (
  <div>
    <h1>{post}</h1>
  </div>
)

export default Post

export async function getStaticPaths() {
  return {
    paths: [
      { params: { id: '1' } },
      { params: { id: '2' } }
    ],
    fallback: false,
  };
}
```

The error message I received was - 

<img width="958" alt="Screenshot 2020-06-06 at 9 24 06 PM" src="https://user-images.githubusercontent.com/22813027/83948630-0f837900-a83c-11ea-9e6f-85571dcb5617.png">



2. Exported `getStaticProps` but **not** `getStaticPaths`. The name of the file was `post/[id].jsx`

```js


const Post = ({ post }) => (
  <div>
    <h1>{post}</h1>
  </div>
)

export default Post

export async function getStaticProps({ params }) {
  return { props: { post: `Example Post ${params.id}` } }
}

```

The error message I received was - 

<img width="960" alt="Screenshot 2020-06-06 at 9 25 53 PM" src="https://user-images.githubusercontent.com/22813027/83948664-4f4a6080-a83c-11ea-9e7b-eca68d13de01.png">


In both the cases, `Next.js` is displaying the correct error.
